source/linear_elasticity.cpp

```diff
@@ -372,8 +372,16 @@ namespace Solid
     auto cell = dof_handler.begin_active();
     auto scalar_cell = scalar_dof_handler.begin_active();
     std::vector<types::global_dof_index> dof_indices(scalar_fe.dofs_per_cell);
+
+    // containers for cell-wise stress output
+    cellwise_sxx.reinit(triangulation.n_active_cells());
+    cellwise_sxy.reinit(triangulation.n_active_cells());
+    cellwise_syy.reinit(triangulation.n_active_cells());
```

Comment:
Need to avoid repeated code like this. For example, when declare them you can use a ``std::vector`` to store all the stresses and operating on them can be done in a loop.

Reply:
Fixed